README.md

```diff
@@ -58,15 +58,29 @@ MessageBus.publish "/channel", "hello", user_ids: [1,2,3], group_ids: [4,5,6]
 # messages can be targetted at particular clients (using MessageBus.clientId)
 MessageBus.publish "/channel", "hello", client_ids: ["XXX","YYY"]
 
-# message bus determines the user ids and groups based on env
+# MessageBus determines the user ids and groups based on env. How that data is calculated is entirely up to you.
 
 MessageBus.configure(user_id_lookup: proc do |env|
-  # return the user id here
+  # Supply the method body for extracting and returning a unique user id.
+
+  # Example: Returning a user_id cookie from the request.
+  request = Rack::Request.new(env)
+  request.cookies["user_id"]
```

Comment:
Can we query database and retrieve post params from inside user_id_lookup ? Something like:
```
req = Rack::Request.new(env)
if  SocialUser.validate_token(req.params["user_id"], req.params["user_token"])
   req.params["user_id"]
else
   nil
end
```